automation/Extension/firefox/lib/loggingdb.js

```diff
@@ -7,18 +7,27 @@ var visitID = null;
 var debugging = false;
 var sqliteAggregator = null;
 var ldbAggregator = null;
+var logAggregator = null;
 var listeningSocket = null;
 
-exports.open = function(sqliteAddress, ldbAddress, crawlID) {
-    if (sqliteAddress == null && ldbAddress == null && crawlID == '') {
+exports.open = function(sqliteAddress, ldbAddress, logAddress, curr_crawlID) {
```

Comment:
The `curr_crawlID` never changes for the lifetime of a browser (and thus an extension instance), can we revert this name change back to `crawlID`?